### DISCUSSION TITLE
Is it possible to add custom blocks to Scratch?

### DISCUSSION THREAD

**Discussion by Kabanter** [ScratchAddons/ScratchAddons#7856] [Help] [answered]
Hi Scratchers, I wanted to ask if it's possible to add custom blocks to Scratch (more specifically the Machine Learning for Kids ones). A quick query on Google Gemini told me it was possible, but I can't figure out how to make it work, I'm really interested in using their neural network feature.

**Comment by WorldLanguages** [comment, marked as answer]
It's not possible to add custom blocks to Scratch itself, I think
